src/components/GPUDashboard/Cards/GPUDashboardGraphs.tsx

```diff
@@ -32,7 +31,7 @@ import { useTranslation } from '../../../i18n';
 */
 
 /* Used metrics
-    DCGM_FI_PROF_GR_ENGINE_ACTIVE, gauge, gpu utilization.
```

Comment:
II{U,R}C we chose `DCGM_FI_PROF_GR_ENGINE_ACTIVE` because it is available also [when using MIG](https://github.com/NVIDIA/DCGM/issues/80#issuecomment-1537603016). 

If we would like to drop MIG support for other NVIDIA GPU models, then no objection from my part (whatever makes more sense business-wise).